src/docs/markdown/caddyfile/directives/encode.md

```diff
@@ -31,7 +31,7 @@ encode [<matcher>] <formats...> {
 
 	match {
 		status <code...>
-		header <field> [<value>]
+		header <field> <value>
```

Comment:
I might be reading this wrong, but this seemed to suggest that the header value is optional. This change to emphasize that both the header name and the value are required.

Reply:
It's technically optional because you can do `-Field` to delete a header, in which case no value is passed.

Reply:
Are you talking about the header directive?

Reply:
Oh, yeah sorry. I didn't click through to the files changed so I didn't notice it was the response matcher in `encode`. That said, `!` as a prefix is allowed, meaning "this header doesn't exist in the response" and in that case you omit the value. See https://caddyserver.com/docs/caddyfile/response-matchers#header

Reply:
Thanks, didn't know about the `!` syntax. I'll revert.